primitives/src/runtime_api.rs

```diff
@@ -220,18 +220,18 @@ sp_api::decl_runtime_apis! {
 		fn session_executor_params(session_index: SessionIndex) -> Option<ExecutorParams>;
 
 		/// Returns a list of validators that lost a past session dispute and need to be slashed.
-		#[api_version(5)]
+		/// NOTE: This function is only available since parachain host version 5.
 		fn unapplied_slashes() -> Vec<(SessionIndex, CandidateHash, vstaging::slashing::PendingSlashes)>;
 
 		/// Returns a merkle proof of a validator session key.
-		#[api_version(5)]
+		/// NOTE: This function is only available since parachain host version 5.
```

Comment:
It's not ideal loosing version information when bumping base but it is what it is :(